src/bp/admin/ui/src/app/translations/en.json

```diff
@@ -306,6 +306,7 @@
           "error": "error",
           "export": "Export",
           "languageIsMissing": "Bot language is missing. Please set it in bot config.",
+          "languageIsNotSupported": "Bot language is not supported by our NLU system.",
```

Comment:
Remove the `languageIsNotSupported`

Modified `LanguageIsMissing`, to pass parameters language that is not supported

Reply:
`Your bot is using a language that is not enabled in the NLU system ${Language String}`